src/app/send/send.component.html

```diff
@@ -107,7 +107,7 @@ <h1>
                                         {{'copySendLink' | i18n}}
                                     </a>
                                     <a class="dropdown-item" href="#" appStopClick (click)="removePassword(s)"
-                                        *ngIf="s.password">
+                                        *ngIf="s.password && !disableSend">
```

Comment:
Not sure whats happening here. It seems to have picked up a unrelated change? https://github.com/bitwarden/web/blob/master/src/app/send/send.component.html#L110 seems to already have this change.

Reply:
merge resolution more than likely, should be okay.